Runtime/Code/Player/Character/MovementSystems/Character/CharacterMovement.cs

```diff
@@ -209,6 +223,7 @@ private void Awake() {
                 physics = new CharacterPhysics(this);
             }
 
+            _rig = GetComponentInChildren<CharacterRig>();
```

Comment:
Would be faster if we had a serialized reference but I can see why you wouldn't want to depend on a serialized property because existing character variants struggle to pick it up sometimes. 

Maybe have this be a serialized and we call `GetComponentInChildren()` if it's null?